README.md

```diff
@@ -1 +1,637 @@
-# spree-storefront-api-v2-js-sdk
+# Spree Storefront API SDK
+
+Node module used for integration front-end with Spree API V2
+---
+
+
+
+Сontents:
+
+- [Quick Start](#quick-start)
+- [Example](#example)
+- [Endpoints](#endpoints)
+  - [Authentication](#authentication)
+    - [getToken](#getToken)
+    - [refreshToken](#refreshToken)
+  - [Account](#account)
+    - [accountInfo](#accountInfo)
+    - [creditCardsList](#creditCardsList)
+    - [defaultCreditCard](#defaultCreditCard)
+    - [completedOrdersList](#completedOrdersList)
+    - [completedOrder](#completedOrder)
+
+
+<br/>
+
+## Quick Start
+
+1. Install SDK
+
+  `npm install spree-storefront-api-v2-js-sdk --save`
+
+<br/>
+
+2. Initialize `Client`
+
+  ```ts
+    import { Client } from 'spree-storefront-api-v2-js-sdk'
+
+    const client = Client({
```

Comment:
I think with `new` it looks a little more standard:
```suggestion
    const client = new Client({
```

Reply:
@tniezg `Client` isn't a class

Reply:
I think we should rename the function to `createClient` or `makeClient`. Doesn't have to be in this PR. Uppercase namers are generally reserved for constructors.